test/chi/components/header.pug

```diff
@@ -567,45 +517,254 @@ h3 Header with Horizontal Navbar
+        span.chi-header__title.-d--none.-d-sm--flex Application Title
+      .chi-header__start
+        .-d--flex.-d-lg--none
+          button#drawer-trigger-1.chi-button.-icon.-flat.-light.chi-drawer__trigger(data-target='#drawer-1', aria-label='Toggle navigation')
+            .chi-button__content
+              i.chi-icon.-sm--2.icon-menu
+      .chi-header__end
+        .-d--flex.-d-lg--none
+          a#drawer-trigger-2.chi-avatar.chi-drawer__trigger(data-target='#drawer-0', href='#')
+            img(src='../../assets/images/avatar.jpg', alt='avatar')
+        .-d--none.-d-lg--flex
+          a.chi-button.-flat.-light(href='#') Item
+          a.chi-button.-flat.-light.-ml--1(href='#') Item
+    .chi-divider.-inverse.-m--0.-bg--black.-w--100
+    nav#drawer-trigger-3.chi-header__toolbar.-d--flex.-d-lg--none.chi-drawer__trigger(data-target='#drawer-3')
+      .chi-header__start
+        i.chi-icon.icon-map-marker.-mx--1.-ml-lg--0.-text--light
+        span.-text--light Menu
+      .chi-header__end
+        i.chi-icon.icon-chevron-down.-mx--1.-mr-lg--0.-text--light
+// Drawer
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    #drawer-1.chi-drawer.-left.-menu.-position--absolute
+      .chi-drawer__header
+        button.chi-button.-icon.-close(aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+      .chi-drawer__content
+        .-px--2.-text Drawer content here
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    #drawer-0.chi-drawer.-right.-menu.-position--absolute
+      .chi-drawer__header
+        button.chi-button.-icon.-close(aria-label='Close')
+          .chi-button__content
+            i.chi-icon.icon-x
+      .chi-drawer__content
+        .-px--2.-text Drawer content here
+.chi-backdrop.-closed
+  .chi-backdrop__wrapper
+    #drawer-3.chi-drawer.-bottom.-menu.-position--absolute
+      button.chi-button.-icon.-close(aria-label='Close')
+        .chi-button__content
+          i.chi-icon.icon-x
+      .chi-drawer__content
+        .-px--4.-py--2
+          ul.chi-tabs.-vertical.-lg
+            li.-active
+              a(href='#') Item
+            li
+              a(href='#') Item
             li
-              a(href='#') Billing
+              a(href='#') Item
+            li
+              a(href='#') Item
+
+
+
```

Comment:
@sabah1910 can you remove these lines as they don't apply to these examples?

Reply:
@jllr Pushed new changes